### ISSUE TITLE
Conveyor-bridge collapsing with conveyors

### ISSUE THREAD

**Issue by Alstin112** [Anuken/Mindustry#9420]
### Platforms

Windows

### Build

146

## Issue

This topic is simlar with the issue #8011, but i will talk with something more deep than that.

when trying cross a conveyor/conduit with blocks on side with another conveyor/conduit, the bridge intesects.
![image](https://github.com/Anuken/Mindustry/assets/108550823/82839ee4-ddd5-404d-b3e9-73be1f413eb0)

This occurs only in all serpulo conveyors/conduit and in specific way on erekir conduit.

And is not only this bug, i want to mention this minor that anoyes too.
![image](https://github.com/Anuken/Mindustry/assets/108550823/1bde0495-cb97-4a7a-aac0-c0e74bcf4491)
and when pass the conveyor
![image](https://github.com/Anuken/Mindustry/assets/108550823/bd8f1a69-c0ab-435f-b260-9c25a176b78b)

And another case:
![image](https://github.com/Anuken/Mindustry/assets/108550823/8475970b-7fa4-41ba-9dd2-9d8e606ba2af)
goes to:
![image](https://github.com/Anuken/Mindustry/assets/108550823/de4d5a56-1c99-4b52-acc6-c207e3fb9b56)

And the last case:
![image](https://github.com/Anuken/Mindustry/assets/108550823/e60711bb-3ee2-49f8-8dfe-dc95b8fea4b3)

## My Propose

The placement of bridges go in 3 priorities, if the first isn't possible, go to the next.
1. Try to place bridge only in the air, this avoid alot of cases that are the first, second and thirth image.
2. Try to place bridge only in conveyors as the same rotation as the initial conveyor, this will avoid the fourth image.
3. Try to place bigde over any conveyor

After these trys, test if all blocks inside are air, junction, conveyors in same rotation (not inverse) or perpendicular then replace the perpendicular ones with junction, the others with the new conveyor.

(also solves the issue with conveyors crossing bridges)
![image](https://github.com/Anuken/Mindustry/assets/108550823/5cc86c87-5b42-42ae-a78f-fbbd2287ee3f)

Of course, the same apply for conduits

### Steps to reproduce

my english is not good enought to describe, soo there is a gif
![mindbug-conveyor-bridge](https://github.com/Anuken/Mindustry/assets/108550823/4b29d915-d708-40d6-ae93-128b6c568ee7)

### Mods used

none

### Submission

- [X] I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.
- [X] I have searched the closed and open issues to make sure that this problem has not already been reported.

**Comment by JasonP01** [comment]
you shouldve testing this in BE builds first, anuke fixed the other one and likely fixed these as well (or atleast most of them)

**Comment by Alstin112** [comment]
You are right! but i tested now and still have this bug in build BE 24674 (the last util now)
![image](https://github.com/Anuken/Mindustry/assets/108550823/14ee5156-a9ec-413e-b536-d2d8880c34dc)
